File: Server/FreelaAPI/Freela.Domain/Models/Project.cs
using System;

namespace Freela.Domain.Models
{
    public class Project
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int Status { get; set; }
        public int Ranking { get; set; }
        public string Area { get; set; }
        public string DurationTime { get; set; }
        public DateTime CreateDate { get; set; }
        public Company Company { get; set; }
        public IEnumerable<UserProject> UserProjects { get; set; }

    }
}

File: Server/FreelaAPI/Freela.Website/Freela_Website/src/app/models/project.ts
import { Company } from "./company";
import { UserProject } from "./userProject";

export interface Project{
    Id:number ;
    Title:string ;
    CreateDate:string;
    Proposes:number ;
    Description:string ;
    Status:number ;
    Ranking:number ;
    Area:string ;
    ValueToPay:string;
    categories:string;
    contaxt:string;
    Company:Company ;
    UserProject?: UserProject[] ;
}
